core/src/main/java/tachyon/Constants.java

```diff
@@ -64,4 +67,9 @@
    * Version 3 [0.6.0] Add lastModificationTimeMs to inode.
    */
   public static final int JOURNAL_VERSION = 2;
+
+  public static final Class<? extends DataServer> DEFAULT_DATA_SERVER_CLASS =
```

Comment:
IMO this should be in a set of consts that is scoped to worker.  Can you move this inside the worker packages